acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -134,7 +134,7 @@ def ruby_command(host)
       def gem_command(host, type='aio')
         if type == 'aio'
           if host['platform'] =~ /windows/
-            "env PATH=\"#{host['privatebindir']}:${PATH}\" cmd /c gem"
+            "env SSL_CERT_FILE=\"C:/Program Files/Puppet Labs/Puppet/puppet/ssl/cert.pem\" PATH=\"#{host['privatebindir']}:${PATH}\" cmd /c gem"
```

Comment:
We were wondering about that. Do you know what might be the more robust way to get it?